proxy/.dockerignore

```diff
@@ -2,7 +2,5 @@
 #
 # SPDX-License-Identifier: LGPL-2.1-or-later
 
-*
-!entrypoint.sh
-!nginx.conf.template
-!proxy_params
+README.md
+build.sh
```

Comment:
I really prefer the whitelist method to ensure we don't accidentally add the wrong files to any images, and as you've moved everything under `files/` anyway, couldn't you just change this to:

```
*
!files/
```